en/misc.ftl

```diff
@@ -235,10 +265,57 @@ upgrade-for-more-protection = Upgrade for more protection
+# Variables
+#   $url (string) - URL of add-on
+#   $attrs (string) - Inline attributes for the link
+first-time-user-email-how-item-1-subhead-html = Create masks directly on { -brand-name-firefox }, with the <a href="{ $url }" { $attrs }>{ -brand-name-relay } add-on</a>, or on your { -brand-name-relay } dashboard.
+first-time-user-email-how-item-1-subhead-text = Create masks directly on { -brand-name-firefox }, with the { -brand-name-relay } add-on, or on your { -brand-name-relay } dashboard.
+first-time-user-email-how-item-2-header = We’ll forward all emails to your inbox
+first-time-user-email-how-item-2-subhead =  Senders will never see your real address, and you can block emails any time.
+first-time-user-email-how-item-3-header = Manage your masks from your { -brand-name-relay } dashboard
+# Variables
+#   $url (string) - URL of the dashboard
+#   $attrs (string) - Inline attributes for the link
+first-time-user-email-how-item-3-subhead-html =  <a href="{ $url }" { $attrs }>Sign in</a> to create new masks, label your masks, and delete masks that get spam.
+first-time-user-email-how-item-3-subhead-text =  Sign in to create new masks, label your masks, and delete masks that get spam.
+
+first-time-user-email-extra-protection-inbox-title = Extra protection for your inbox
+first-time-user-email-extra-protection-inbox-phone-title = Extra protection for your inbox and phone
+first-time-user-email-extra-protection-inbox-subhead = Upgrade to { -brand-name-relay-premium } to get unlimited email masks, a custom { -brand-name-relay } domain, and custom inbox controls.
+first-time-user-email-extra-protection-inbox-phone-subhead = Upgrade to { -brand-name-relay-premium } for unlimited email masks — plus a phone mask to protect your real number.
+first-time-user-email-extra-protection-cta = Get { -brand-name-relay-premium }
+
+first-time-user-email-questions-title = Questions about { -brand-name-firefox-relay }?
+# Variables
+#   $url (string) - URL of the support team website
+#   $attrs (string) - In-line attributes for the link
+first-time-user-email-questions-subhead-html = Our <a href="{ $url }" { $attrs }>support team</a> is here to help.
+first-time-user-email-questions-subhead-text = Our support team is here to help.
+first-time-user-email-footer-text-1 = You’re receiving this automated email as a subscriber of { -brand-name-firefox-relay } that used { -brand-name-relay } for the first time. If you received it in error, no action is required.
+# Variables
+#   $url (string) - URL of the support team website
+#   $attrs (string) - In-line attributes for the link
+first-time-user-email-footer-text-2-html = For more information, please visit <a href="{ $url }" { $attrs }>{ -brand-name-mozilla } Support</a>.
+first-time-user-email-footer-text-2-text = For more information, please visit { -brand-name-mozilla } Support.
+first-time-user-email-footer-text-address = 2 Harrison St. #175, San Francisco, California 94105 USA
```

Comment:
What's the purpose of having a physical address listed here? I think the only  time I see an address is our legal documents.

Reply:
It is in the original email design, but I agree, it is interesting that we have it. There's nothing to translate, so I'll remove it.